src/main/java/alanExceptions/AlanException.java

```diff
@@ -0,0 +1,7 @@
+package alanExceptions;
+
+public class AlanException extends Exception{
```

Comment:
Don't forget to place a space between { and Exception 😬 
Found this in another part as well, might wanna look through.